mantidproject/mantid
# Release notes link in first time setup using qthelp

The release notes link in the First time setup GUI used to use the docs.mantidproject webpage.
This could mean that the release notes might no longer apply to that version, now we have bundled release notes with qtHelp we can match versions exactly, even for nightly builds.


**To test:**

1. Start mantidplot
1. open Help->first time setup
1. Click on the Release notes
1. They should appear in the qt help browser
   - Dev builds should point at the next releases notes
   - Official releases should point at there own.

Fixes #18773.

### RELEASE NOTES

*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
